src/Analysis/Ast/Impl/Analyzer/Handlers/FromImportHandler.cs

```diff
@@ -82,7 +82,7 @@ internal sealed partial class ImportHandler {
             }
 
             // If __all__ is present, take it, otherwise declare all members from the module that do not begin with an underscore.
-            var memberNames = variableModule.Analysis.StarImportMemberNames ?? variableModule.GetMemberNames().Where(s => !s.StartsWithOrdinal("_"));
+            var memberNames = variableModule.Analysis?.StarImportMemberNames ?? variableModule.GetMemberNames().Where(s => !s.StartsWithOrdinal("_"));
```

Comment:
If `variableModule` represents implicit package, `Analysis` will be `null`.

Reply:
Kinda dangerous... Maybe have it attach empty analysis?

Reply:
Empty analysis requires `IDocument` but there is actually no document - it is an implicit package.

Reply:
OK